flake.nix

```diff
@@ -133,6 +135,15 @@
               src = self;
               exe = "${packages.plutus-merkle-tree-tests}/bin/tests";
             };
+            hydra-cardano-api-werror = addWerror hsPkgs.hydra-cardano-api.components.library;
+            hydra-cluster-werror = addWerror hsPkgs.hydra-cluster.components.library;
+            hydra-chain-observer-werror = addWerror hsPkgs.hydra-chain-observer.components.exes.hydra-chain-observer;
+            hydra-plutus-werror = addWerror hsPkgs.hydra-plutus.components.library;
+            hydra-plutus-tests-werror = addWerror hsPkgs.hydra-plutus.components.exes.tests;
+            hydra-node-werror = addWerror hsPkgs.hydra-node.components.library;
+            hydra-node-tests-werror = addWerror hsPkgs.hydra-node.components.tests.tests;
+            hydra-tui-werror = addWerror hsPkgs.hydra-tui.components.library;
+            hydra-tui-tests-werror = addWerror hsPkgs.hydra-tui.components.tests.tests;
```

Comment:
How did you pick which to include and which not to? AFAIS these are not all [local packages](https://github.com/input-output-hk/hydra/blob/d8d2f93ae1c5adafc71b0264b01b7d4999751f78/cabal.project#L19-L32), but also not all `hydra-*` prefixed local packages.

Reply:
This was just haste actually. It is unfortunate that we can not just map over the names of the packages, but seemingly we need to account for each subcomponent as well. I just picked /a bunch/. Should we try and derive the attribute set?

Reply:
OK now it maps all components.